contracts/modules/cover/Gateway.sol

```diff
@@ -28,7 +28,7 @@ import "../token/TokenFunctions.sol";
 import "./QuotationData.sol";
 import "../claims/ClaimsReward.sol";
 
-contract Cover is MasterAware {
+contract Gateway is MasterAware {
```

Comment:
tests naming, and actual deployment aside

there's one place where the Gateway is referred. to by code here:

https://github.com/NexusMutual/distributor-contract/blob/3f4f9f9823cdee2466b4eda2d73ab0b4c77fbf9d/contracts/DistributorFactory.sol#L30


we'd need to update that at the very least to make it compatible with the new "CO" -> "GW"